tools/scripts/ansible/setup-playbook.yml

```diff
@@ -52,25 +61,69 @@
         force: no
         owner: tomcat8
         group: tomcat8
-        mode: 0644
+        mode: "0644"
       notify: Restart Tomcat
     - name: Checkout CLDR trunk
       git:
         repo: https://github.com/unicode-org/cldr.git
         dest: /var/lib/tomcat8/cldr/cldr-trunk
-        depth: 1 # don't need all history
         force: no
+        update: no
         version: master
+        # this is deep because we will need to keep updating
+        # it with history. It does not include LFS as that
+        # is not needed for the surveytool.
+    - name: Setup index.html
+      copy:
+        src: templates/index.html
+        dest: /var/www/html
+        owner: root
+        group: root
+        mode: '0644'
+    - name: Setup reverse proxy
+      blockinfile:
+        path: /etc/nginx/sites-enabled/default
+        block: |
+          # proxy /cldr-apps/ to tomcat
+          location /cldr-apps/ {
+            allow all;
+            proxy_pass http://localhost:8080/cldr-apps/;
+            proxy_set_header Host $host;
+            proxy_set_header X-Real-IP $remote_addr;
+          }
+        marker: '# {mark} ANSIBLE MANAGED BLOCK'
+        insertafter: '^[\s]*server_name' # the LAST uncommented server block
+      notify: 'Restart Nginx'
+    - name: Setup surveytool user for deploy
+      user:
+        name: surveytool
+        shell: /bin/bash
+    - name: Give acess to surveytool user
+      file:
+        path: /var/lib/tomcat8/cldr/cldr-trunk
+        owner: surveytool
+        recurse: yes
+    - name: Setup surveytool auth
```

Comment:
The GH Action for Ansible build is currently failing with an error saying that there's an invalid key name in the YAML around somewhere here.

Reply:
@echeran please recheck… think i fixed it